bfdrivers/src/arch/osx/fixup-kext-perms.sh

```diff
@@ -0,0 +1,5 @@
+#!/bin/bash
+
+sudo chown -R root /Users/brendank310/Library/Developer/Xcode/DerivedData/bareflank-*/Build/Products/Debug/bareflank.kext
```

Comment:
I'd argue yes, though I think I may have updated the unfixup-kext-perms and not this to remove the hardcoded path. Without it, you cannot load up the kext. You would think XCode would change the perms after a build, but it currently does not. I don't know a lot about XCode but perhaps the project itself can have a pre/post build hook to change the permissions.